rfc3164/machine.go.rl

```diff
@@ -141,6 +165,23 @@ rfc3339 = fulldate >mark 'T' hhmmss timeoffset %set_rfc3339 @err(err_rfc3339);
 # note > this could mean that the we may need to create and to use a labelrange = graph{1,63} here if we want the parser to be stricter.
 hostname = (hostnamerange -- ':') >mark %set_hostname $err(err_hostname);
 
+# Cisco IOS devices sometimes include a "message counter" before the timestamp
+# "<189>237: *Jan  8 19:46:03.295..."
+msgcountval = (digit*) >mark %set_msgcount @err(err_msgcount);
```

Comment:
Why do we have `digit*` here while for the sequence number we do have `digit+`?

Asking because it would parse `<PRI>:` as valid, but what would be the message counter value? Zero?

Reply:
Yeah, that's what happens if you disable the message counter: https://github.com/leodido/go-syslog/pull/22/files#diff-7c1da6249183aa35b9f43da53d4df1d7bfa91d32bc29e63267436cf45b7686cdR298-R299

See also this comment https://github.com/leodido/go-syslog/pull/22#discussion_r1990756563